app/pubspec.yaml

```diff
@@ -61,7 +61,11 @@ dependencies:
       ref: main
 
   flutter_easyloading: ^3.0.5
-  appflowy_editor: ^5.2.0
+  appflowy_editor: #^5.2.0
```

Comment:
is something also internally changed within plugin ? An issue within ```appflowy``` ? Or a workaround for our case?

Reply:
yes. as the mobile toolbar is using `Column` internally, but our expandable editor is within a valuelistener, that doesn't report a size by default but is within a `columns` `expanded` there is competing interest in what to do: squeeze or expand. [The change in the fork](https://github.com/AppFlowy-IO/appflowy-editor/compare/main...acterglobal:appflowy-editor:ben-mobile-expandable-editor) fixes that problem by using `Flexible` rather than `Expanded` and inform the `Column` to squeeze in size if not otherwise noted. That fix is needed for our editor to work on mobile as well.

Reply:
I am not yet sure, if that is generic enough to sent upstream, but I will make a PR and discuss that with the team.